tests/yast2_gui/yast2_users.pm

```diff
@@ -16,11 +16,107 @@ use strict;
 use warnings;
 use testapi;
 
+use utils qw(type_string_slow_extended);
+use version_utils qw(is_tumbleweed is_leap);
+
+sub write_change_now {
+    send_key 'alt-x';
+    send_key 'down';
+    send_key 'down';
+    send_key 'down';
+    send_key 'ret';
+}
+
+sub check_with_xterm {
+    my $command = shift;
+    x11_start_program('xterm');
+    become_root;
+    assert_script_run($command);
+    wait_screen_change { send_key 'alt-f4'; };
+}
+
+sub add_user {
+    my (%args) = @_;
+    send_key 'alt-a';    # add user tab
+    type_string_slow_extended($args{username});
+    send_key 'tab';
+    type_string_slow_extended($args{username});
+    send_key 'tab';
+    type_string_slow_extended($args{password});
+    send_key 'tab';
+    type_string_slow_extended($args{password});
+    assert_screen 'user_data_fields_filled_up';
+    send_key "alt-o";    # OK => Exit
+    if (is_leap || is_tumbleweed) {
+        wait_still_screen 3;
```

Comment:
This a problem, you will never have still screen when there is cursor flashing with some frequency way smaller than 3 seconds. Therefore your *wait_still_screen* call will always time out.